ssl/ssl_txt.c

```diff
@@ -55,7 +55,7 @@ int SSL_SESSION_print(BIO *bp, const SSL_SESSION *x)
     } else {
         if (BIO_printf
             (bp, "    Cipher    : %s\n",
-             ((x->cipher == NULL) ? "unknown" : x->cipher->name)) <= 0)
+             ((x->cipher->name == NULL) ? "unknown" : x->cipher->name)) <= 0)
```

Comment:
I'd like to say since we are at it, it would be only appropriate to fix formatting as
```
        if (BIO_printf(bp, "    Cipher    : %s\n",
                       ((x->cipher->name == NULL) ? "unknown"
                                                  : x->cipher->name)) <= 0)
```
And there is pair of similarly misformatted calls to BIO_printf just above this one. You're free to decline, and lack of response will be viewed as [totally acceptable] refusal...

Reply:
Thanks for the review, I tried to incorporate your suggestions